src/main/java/main/Twitch4J/Twitch4JInput.java

```diff
@@ -1,4 +1,4 @@
-package main.inputs.Twitch4J;
+package main.Twitch4J;
```

Comment:
Twitch4J should be a submodule of inputs, as there will be added more inputs, for example TipeeeStream.

Reply:
Yeah, please keep the original folder structure intact.
All Inputs should be in the Inputs directory and implement the TwitchBotInput Interface.

More generally, if someone has questions about implementing something, like me. Then I know what I should include in the wiki articles (that I'm going to write when I have time for that)